internal/rpcserver/rpcserverhandlers_test.go

```diff
@@ -5933,6 +5933,74 @@ func TestHandleTSpendVotes(t *testing.T) {
 	}})
 }
 
+func TestHandleTicketVWAP(t *testing.T) {
+	t.Parallel()
+
+	start := uint32(10)
+	beforeStart := uint32(0)
+	afterBestHeight := block432100.Header.Height + uint32(1)
+	end := uint32(15)
+	vwap := 144.2816259
+
+	testRPCServerHandler(t, []rpcTest{{
+		name:    "handleTicketVWAP: start is greater than end",
+		handler: handleTicketVWAP,
+		cmd: &types.TicketVWAPCmd{
+			Start: &start,
+			End:   &beforeStart,
+		},
+		wantErr: true,
+		errCode: dcrjson.ErrRPCInvalidParameter,
+	}, {
+		name:    "handleTicketVWAP: end is after the current best height",
+		handler: handleTicketVWAP,
+		cmd: &types.TicketVWAPCmd{
+			Start: &start,
+			End:   &afterBestHeight,
+		},
+		wantErr: true,
+		errCode: dcrjson.ErrRPCInvalidParameter,
+	}, {
+		name:    "handleTicketVWAP: unable to fetch header by height",
+		handler: handleTicketVWAP,
+		cmd: &types.TicketVWAPCmd{
+			Start: &start,
+			End:   &end,
+		},
+		mockChain: func() *testRPCChain {
+			chain := defaultMockRPCChain()
+			chain.headerByHeightErr =
+				fmt.Errorf("unable to fetch header by height")
+			return chain
+		}(),
+		wantErr: true,
+		errCode: dcrjson.ErrRPCInternal.Code,
+	}, {
+		name:    "handleTicketVWAP: ok,",
```

Comment:
Looks like a leftover comma left here
```suggestion
		name:    "handleTicketVWAP: ok",
```